joomla/joomla-cms
# [4.0] Remove jQuery dependency for com_joomlaupdate Pre-Update Check

Pull Request for part of Issue #31490.

### Summary of Changes
My attempt to remove jQuery dependency for com_joomlaupdate Pre-Update Check.


### Testing Instructions
1. Login to administrator area of your site. Access to Joomla Update component (System -> Update -> Joomla) with this URL http://localhost/joomla4/administrator/index.php?option=com_joomlaupdate  (replace http://localhost/joomla4/administrator/ with administrator URL of your site)

- Click on **Options** button in the toolbar
- Set **Update Channel** to **Custom URL**
- Set **Minimum Stability** to **Beta**
- Set **Custom URL** to **https://update.joomla.org/core/nightlies/next_major_list.xml**

2. Install an old version of an extension you know that there is new update available. I would suggest you to install this old version of Akeeba Backup extension which can be downloaded from this URL https://www.akeeba.com/download/akeeba-backup/8-0-1/pkg_akeeba-8-0-1-core-zip.zip

3. Get the attached zip file (it contains complied javascript files comes from this PR), unzip it to the folder **media** of your site (it will replace the current javascript code in media/com_joomla update folder)
[com_joomlaupdate.zip](https://github.com/joomla/joomla-cms/files/6135002/com_joomlaupdate.zip)

4. Come back to Joomla Update Screen, press Check for Updates button, confirm that you see a screen like below telling you that there are Joomla update available and extensions compatible with that update.

![joomla_update_pre_check](https://user-images.githubusercontent.com/977664/111034802-dcfd5100-8449-11eb-8e81-bfe20732d369.png)

5. Click on **[ More Detail ]** text, confirm that you will see more for the table Displayed and the text will be changed to **[Less Detail]**

![pre-update-check-more-detail](https://user-images.githubusercontent.com/977664/111034904-4f6e3100-844a-11eb-89f2-1b55972b46f8.png)

6. Click again on **[Less Detail]**, confirm that some columns are hidden in the table and the text changed back to **[More Detail]** like before

![pre-update-check-less-detail](https://user-images.githubusercontent.com/977664/111034976-aaa02380-844a-11eb-9cd4-734b6e28d537.png)
